qaframework-bdd-tests/package.json

```diff
@@ -49,7 +49,7 @@
   "devDependencies": {
     "@testing-library/cypress": "^8.0.0",
     "@types/uuid": "3.4.7",
-    "cypress": "^8.2.0",
+    "cypress": "^9.5.3",
```

Comment:
Is this the latest version of cypress

Reply:
Yeah @sherrif10 this is the stable and latest version of cypress.